packages/compass-indexes/src/utils/index-link-helper.js

```diff
@@ -13,6 +13,7 @@ const HELP_URLS = {
   GEOSPATIAL:
     'https://docs.mongodb.org/manual/applications/geospatial-indexes/#geospatial-indexes',
   WILDCARD: 'https://docs.mongodb.com/manual/core/index-wildcard/',
+  // TODO: add an entry for CLUSTERED once the docs become available
```

Comment:
A similar comment to the one in the first pr - https://github.com/mongodb-js/compass/pull/2966#discussion_r854479425
Do we have a ticket name we can attach to this comment? I find it can help give context quickly when looking through code (and then if we somehow miss updating this todo we know way down the line it can be quickly updated).

Reply:
https://jira.mongodb.org/browse/COMPASS-5760

And I linked to it from both places